src/server/detail/snapshot_storage.h

```diff
@@ -0,0 +1,110 @@
+// Copyright 2023, DragonflyDB authors.  All rights reserved.
+// See LICENSE for licensing terms.
+
+#pragma once
+
+#include <filesystem>
+#include <string>
+#include <string_view>
+#include <utility>
+
+#include "io/io.h"
+#include "server/common.h"
+#include "util/cloud/aws.h"
+#include "util/fibers/fiberqueue_threadpool.h"
+#include "util/uring/uring_file.h"
+
+namespace dfly {
+namespace detail {
+
+namespace fs = std::filesystem;
+
+using namespace std::literals;
```

Comment:
we tend to avoid calling "using namespace" in header files.

Reply:
that seems to be needed for the `"s3://"sv` literal (not sure if theres a way to do it without `using`?)

Reply:
You don't need it there, it works implicitly. You only need the sv prefix if you want to differentiate between const char* and string_view when passing to a function that might accept both, etc

Reply:
Ok cool thanks - removed